src/index/src/inverted_index/create/sort.rs

```diff
@@ -17,22 +17,24 @@ mod intermediate_rw;
 mod merge_stream;
 
 use async_trait::async_trait;
-use common_base::BitVec;
 use futures::Stream;
+use roaring::RoaringBitmap;
 
+use crate::bitmap::Bitmap;
 use crate::inverted_index::error::Result;
+use crate::inverted_index::format::writer::ValueStream;
 use crate::{Bytes, BytesRef};
 
 /// A stream of sorted values along with their associated bitmap
-pub type SortedStream = Box<dyn Stream<Item = Result<(Bytes, BitVec)>> + Send + Unpin>;
+pub type SortedStream = Box<dyn Stream<Item = Result<(Bytes, RoaringBitmap)>> + Send + Unpin>;
```

Comment:
why it's not `Bitmap` but a concrete `RoaringBitmap`?

Reply:
Well, using `Bitmap` is also good, I'll change it.